x/oracle/types/event.go

```diff
@@ -0,0 +1,8 @@
+package types
+
+const (
+	EventTypeRegisterOracle = "register"
+
+	AttributeKeyOracle   = "oracle"
+	AttributeValueOracle = "RegisterOracleEvent"
```

Comment:
How about changing it like below?
It looks like we can use EventType and AttributeKey in common.

```suggestion
	EventTypeVote= "vote"
	AttributeKeyVoteType   = "voteType"

	VoteTypeRegisterOracle = "RegisterOracle"
	VoteTypeRegisterOracleComplete = "RegisterOracleComplete"
```

Reply:
I think you can use this VoteType like an enum.
```
type VoteType string

const (
	VOTE_TYPE_REGISTER_ORACLE VoteType = "OracleRegister"
	VOTE_TYPE_REGISTER_ORACLE_COMPLETE VoteType = "OracleRegisterComplete"
)
```

Reply:
Fixed in [2e806ae](https://github.com/medibloc/panacea-core/pull/382/commits/2e806ae059cdeabdd6fcaad7ecb7b96792f27c9e). Plus, I didn't use `VoteType`. If I add the `VoteType` the cosmos-sdk said that convert to string.

Reply:
From cosmos-sdk, it seems that naming convention is snake_case.
https://github.com/cosmos/cosmos-sdk/blob/eef086f77071e7ccd68e808aa5460de84ecbc387/x/bank/types/events.go#L18

Reply:
@gyuguen @inchori
Hm.. I think the previous naming made by @inchori is better. 
Does this event present that an oracle was just registered now? Then, I think it's better to emit an event that "A new oracle was registered!!!". 
Then, the next action (voting) should be decided by subscribers.
We don't need to explicitly specify the next action in the event.

Reply:
Yes, I understood what @youngjoon-lee was saying.

Then we have some Event handling to implement in the future.
* oracle registration requsted
* oracle registration vote ended
* data sell requested
* ...

Are we going to put these behaviors in the EventType?

Reply:
Yes. I think so. It's similar as what the `x/bank` does. When coins are transferred between accounts, one of events that it emits is:
- event type: `coin_received`
- attr
  - key: `receiver`
    value: `panacea1qewori....`
  - key: `amount`
    value: `100umed`

Reply:
Okay, I understand it. For now, I'll make that the attribute key value of oracle registration event is address of oracle.

Reply:
Aha, so the subscriber will have no problem subscribing to this event, right?

Reply:
I thought the subscriber should know all this format `{eventType}.{attributeKey}={attributeValue}`

Reply:
Ah, I understand @gyuguen 's comment now. How to subscriber know the `attributeValue`? The subscriber use `query` but they already need to know `attributeValue`.

Reply:
Yes, If the subscriber can detect this Event by its eventType then it should be fine.

Reply:
Okay, for now I apply these comments. And I'll test on `panacea-doracle`.

Reply:
Fixed in [3bc2bac](https://github.com/medibloc/panacea-core/pull/382/commits/3bc2bacc415dac91757b00d03f7c351147c0340f).